### PR TITLE
Add docs for the batch mode of plugin installation

### PR DESCRIPTION
**Pull request by clintongormley**
Closes https://github.com/elastic/elasticsearch/issues/21490

### CODE REVIEW

File: docs/plugins/plugin-script.asciidoc [hunk 1]
```diff
@@ -127,6 +127,24 @@ return the following exit codes:
 `70`:: any other error
 
 [float]
+=== Batch mode
+
+Certain plugins require more privileges than those provided by default in core
+Elasticsearch. These plugins will list the required privileges and ask the
+user for confirmation before continuing with installation.
```
Comment (jasontedor) [inline docs/plugins/plugin-script.asciidoc]: Is it worth being explicit that if you run in batch mode you automatically grant all requested permissions?

**Review by jasontedor — COMMENTED** [review]
I left a question.

**Review by jasontedor — APPROVED** [review]
LGTM.

**Comment by clintongormley** [comment]
thanks for the review @jasontedor

**Pull request merged**